release_notes_generator/model/record.py

```diff
@@ -77,6 +78,15 @@ def pulls(self) -> list[PullRequest]:
         """
         return self.__pulls
 
+    @property
+    def commits(self) -> dict:
+        """
+        Gets the commits associated with the record.
+
+        :return: The commits associated with the record.
```

Comment:
I am not sure, if it is part of current PR, but since we agree to using @return, @param way how to write method docstrings, I would keep on that. On the other hand. Getters are special as we spoke, you don't have to write @return statement, if it doesn't have any extra logic in it. Decide your way how to work with getters in the project, but keep it everywhere same.

Reply:
This is not part of this PR but the following one, after introducing the black tool. Please ignore the format and presence of doc strings in this PR.